From e0807ef3f52f630dcfa694802273d63bbfd35f15 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Wed, 4 Apr 2018 18:48:27 +0300
Subject: [PATCH] ARM64: dts: renesas: ulcb: Make AK4613 sound device name
 consistent

This makes AK4613 sound device name consistent throughout all
ULCB boards. This helps to avoid naming issues when pulseaudio
fails to start because the device has different names on
infotainment and starter boards. For example, "ak4613"
on Kingfisher vs "rsnd-dai.0-ak4613-hifi" on Starter Kit.
Pulseaudio is expecting "ak4613" name.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts | 1 +
 arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts     | 1 +
 arch/arm64/boot/dts/renesas/r8a7796-m3ulcb.dts     | 1 +
 3 files changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
index bffa775..124379d 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
@@ -178,6 +178,7 @@
 		compatible = "simple-audio-card";
 
 		simple-audio-card,format = "left_j";
+		simple-audio-card,name = "ak4613";
 		simple-audio-card,bitclock-master = <&sndcpu>;
 		simple-audio-card,frame-master = <&sndcpu>;
 
diff --git a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
index 76da9f2..4a94a68 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
@@ -178,6 +178,7 @@
 		compatible = "simple-audio-card";
 
 		simple-audio-card,format = "left_j";
+		simple-audio-card,name = "ak4613";
 		simple-audio-card,bitclock-master = <&sndcpu>;
 		simple-audio-card,frame-master = <&sndcpu>;
 
diff --git a/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb.dts
index 7b58109..15c67ec 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb.dts
@@ -159,6 +159,7 @@
 		compatible = "simple-audio-card";
 
 		simple-audio-card,format = "left_j";
+		simple-audio-card,name = "ak4613";
 		simple-audio-card,bitclock-master = <&sndcpu>;
 		simple-audio-card,frame-master = <&sndcpu>;
 
-- 
2.7.4

